web/src/router.rs

```diff
@@ -445,27 +504,43 @@ mod organization_endpoints_tests {
                 .text()
                 .await?;
 
-            assert_eq!(response, json!(organization6).to_string());
+            // We need to parse the values to serde_json::Value to compare them
+            // // so that the attribute order does not matter.
+            let parsed_response: serde_json::Value = serde_json::from_str(&response_text).unwrap();
+
+            assert_eq!(parsed_response, json!(organization6));
         }
 
         Ok(())
     }
 
-    // Purpose: adds multiple Organization instances to a mock DB and tests that calling
-    // the appropriate endpoint updates an instance specified by an ID.
+    // // Purpose: adds multiple Organization instances to a mock DB and tests that calling
+    // // the appropriate endpoint updates an instance specified by an ID.
```

Comment:
```suggestion
    // the appropriate endpoint updates an instance specified by an ID.
```

Reply:
updated in 3d201b56bb1b97fe95447580f25b3ce9ba5afb7f!